L/libgifextra/build_tarballs.jl

```diff
@@ -0,0 +1,50 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "Giflibextra"
+version = v"0.0.1"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://github.com/ashwani-rathee/io-project/raw/main/gifwrapper/gen/libgifextra.zip", "4e1d3e0d1e9df0f30e0dc6c907bb2d574d4c4efec79fb1558451d4c24fd3430b")
```

Comment:
We need to make this a standalone git repository first; the `io-project` repository is only for experimental purposes.

Reply:
Yes, a standalone repository would be nicer.  Also, _**never**_ reference a branch name, like `main`, in the URL: it's a moving target, if you push another commit to the repository this recipe won't work anymore, which isn't acceptable.